docs/_source/tutorials/notebooks/training-textclassification-setfit-sentiment.ipynb

```diff
@@ -0,0 +1,322 @@
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "68306dacccdb45f598436446901d5bd1",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Epoch:   0%|          | 0/1 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "1a978e093a2644798f0c658fe39c5a51",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Iteration:   0%|          | 0/540 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "{'accuracy': 0.8611111111111112}"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model = SetFitModel.from_pretrained(\"sentence-transformers/paraphrase-mpnet-base-v2\")\n",
+    "\n",
+    "# Create trainer\n",
+    "trainer = SetFitTrainer(\n",
+    "    model=model,\n",
+    "    train_dataset=labelled_ds[\"train\"],\n",
+    "    eval_dataset=labelled_ds[\"test\"],\n",
+    "    loss_class=CosineSimilarityLoss,\n",
+    "    batch_size=8,\n",
+    "    num_iterations=20,\n",
+    ")\n",
+    "\n",
+    "trainer.train()\n",
+    "metrics = trainer.evaluate()\n",
+    "metrics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "449b659e-ebd4-4488-8f7f-7848e02fe09a",
+   "metadata": {},
+   "source": [
+    "Here we are using the simplest approach for training our SetFit model. Since it [integrates with Optuna](https://github.com/huggingface/setfit/blob/main/notebooks/text-classification_hyperparameter-search.ipynb), you could use hyperparameter tuning to find the best hyperparamaters for training your model. However, it is better to start with a simple baseline, validate the model for your use case and iterate on the data before focusing on model experimentation and tuning."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "529d3e6b-0684-47db-ab69-ff705945363f",
+   "metadata": {},
+   "source": [
+    "## Summary\n",
+    "\n",
+    "In this tutorial, you learned how to build a training set from scratch and train a sentiment classifier for your own problem.\n",
+    "\n",
+    "Although this is a simple example, you can apply this same process to your own use case. \n",
+    "\n",
+    "Here, we've covered one way of building training sets: **hand labeling**.\n",
+    "\n",
+    "If want to discover other methods like weak supervision or active learning check the following tutorials:\n",
+    "\n",
+    "- [Building a news classifier with weak supervision](labelling-textclassification-sentencetransformers-weaksupervision.ipynb)\n",
+    "- [Active learning with small-text](training-textclassification-smalltext-activelearning.html)"
```

Comment:
It may be worthwhile to also link to the other SetFit tutorial (and vice versa!) for those users interested in using SetFit with Argilla.

Reply:
Agree!